ops/packaging/Dockerfile

```diff
@@ -94,7 +94,7 @@ ARG INSTALL_XDEBUG=false
 RUN if [ ${INSTALL_XDEBUG} = true ]; then \
   # Install the xdebug extension
   if [ $(php -r "echo PHP_MAJOR_VERSION;") = "5" ]; then \
-    pecl install xdebug-2.5.5; \
+    pecl install xdebug-2.9.8; \
```

Comment:
@kencho51 I'm not sure why version `2.9.8` is specified here as the same version number is also below?

Reply:
@pli888 I just want to make sure only xdebug-`2.9.8` will be installed when building the containers. Or should I remove the `else` block?

Reply:
@kencho51 The top of `Dockerfile` says an image with PHP version 7 will be used when building containers with `Dockerfile`. Based on the conditional statements then `xdebug-2.9.8` is installed. You can check this by dropping into the `test` container and checking the version of xdebug installed:
```
$ docker-compose run --rm test bash
Creating deployment_test_run ... done
root@bf59cce4c0cb:/var/www# php -v
PHP 7.1.30 (cli) (built: Jun 11 2019 04:18:15) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.1.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.1.30, Copyright (c) 1999-2018, by Zend Technologies
    with Xdebug v2.9.8, Copyright (c) 2002-2020, by Derick Rethans

root@bf59cce4c0cb:/var/www# pear list
Installed packages, channel pecl.php.net:
=========================================
Package   Version State
apcu      5.1.8   stable
libsodium 2.0.23  stable
mailparse 3.1.1   stable
xdebug    2.9.8   stable
```